### ISSUE TITLE
improvement of the pre-training - test

### ISSUE THREAD

**Issue by simonmeoni** [simonmeoni/bert-impl#64]
* test if [PAD, PAD, PAD, token, PAD] as a target improve the training
* use gelu normalization instead

**Comment by simonmeoni** [comment]
* some attempt during #67 but the loss scaled decreased but it stills too high
* I will do another try after the issue #74

**Comment by simonmeoni** [comment]
done not the result expected
